src/dune_pkg/fetch.mli

```diff
@@ -1,3 +1,3 @@
 open Stdune
 
-val fetch : OpamUrl.t -> target:Path.t -> unit Fiber.t
+val fetch : Loc.t -> OpamUrl.t -> target:Path.t -> unit Fiber.t
```

Comment:
What exactly is the loc meant to represent here? Is it the location in the dune file the fetch will come from? If so can you doc.

Reply:
Yes, I strongly assume so after reading some other PRs.